src/app/profile/certificates/components/certificateTable/CertificateTable.tsx

```diff
@@ -0,0 +1,113 @@
+'use client';
+
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from '@/components/ui/table';
+import { CrossIcon, UniversityIcon } from 'lucide-react';
+import { FC, useCallback, useEffect, useState } from 'react';
+
+import { Button } from '@/components/ui/button';
+import {
+  deleteCertificate,
+  downloadCertificate,
+  fetchCertificates,
+} from '../../services/certificates-service';
+import { useUserSession } from '@/services/auth/useUserSession';
+import { type CertificateModel } from '../../models/certificate.model';
+import { CertificateTypeEnum } from '../../enums/certificate-types.enum';
+import { CERTIFICATE_TRANSLATION } from '../../constants/certificate-translation';
+
+export const resolveIcon = (type: CertificateTypeEnum) => {
```

Comment:
Do we need to export it or is it just local variable?

Reply:
We do not need to export it. Muscle memory to export everything😅